content/en/news/2024/stream-recap-13-oct.org

```diff
@@ -5,27 +5,27 @@
 #+begin_export html
 <iframe width="560" height="315" src="https://www.youtube.com/embed/r_uYPfz4ueY?si=jfrHYbHBFrmi8PQL" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 #+end_export
 * Game Streams
 This weeks ghost cat took over all of my streams! I don't know what her deal is, but hopefully my viewers will be safe! We played Prey, Fatal Frame II, and A Plague Tale again.
 ** Prey
-I got a lot of scares in this week, or rather I got scared a lot this time! I did manage to gain mimic powers. So now I can scare the living daylights out of other now, maybe! Didn't seem to work so well yet, but maybe later! Anyway, got stuck by a phantom, but we'll take care that next time!
+I got a lot of scares in this week, or rather I got scared a lot this time! I did manage to gain mimic powers. So now I can scare the living daylights out of others, maybe! Didn't seem to work so well yet, but maybe later! Anyway, got stuck by a phantom, but we'll take care that next time!
 #+begin_export html
 <iframe width="560" height="315" src="https://www.youtube.com/embed/2cEnFjMyYuk?si=yZ-B9xGyZLSdY88O" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 #+end_export
 ** Fatal Frame II: Crimson Butterfly
-We are still trying to locate sister. I may have spent too much time not quite understanding what I was supposed to do. I think I got it in the end though! Yes, dark came is still dark 😅 Also, the flashlight doesn't seem to work anymore...
+We are still trying to locate sister. I may have spent too much time not quite understanding what I was supposed to do. I think I got it in the end though! Yes, dark game is still dark 😅 Also, the flashlight doesn't seem to work anymore...
 #+begin_export html
 <iframe width="560" height="315" src="https://www.youtube.com/embed/uig9lyMLFCg?si=4DTSqEj35VRYzIn9" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 #+end_export
 ** A Plague Tale: Innocence
-We went to university and got a book! We didn't check it out though, and there may have been a little fire, but...uhm...its fine...I'm sure everything is fine 😅 Also, still rats!
+We went to university and got a book! We didn't check it out though, and there may have been a little fire, but...uhm...its fine...I'm sure everything is fine 😅 Also, still many a rat o:!
```

Comment:
oh thank you, i was thinking i should have that. idk y i didn't right it like that 😅